test/Microsoft.ML.TimeSeries.Tests/TimeSeriesEstimatorTests.cs

```diff
@@ -41,7 +41,7 @@ public TimeSeriesEstimatorTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCore))] // netcore3.0 output differs from Baseline
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCore))] // data is not properly alligned for netcoreapp3.0
```

Comment:
I don't understand the comments on these two tests.

We should change the implementation so it works on any .NET version.

Reply:
The alignment is hardcoded to 16 bit in timeseries code which causes the debug.Assert statements to fail for avx. In avx, these statements check if the data is 32 bit aligned.
I will correct it in the next commit